### ISSUE TITLE
Incorrect bbox

### ISSUE THREAD

**Issue by testerez** [mondeja/svg-path-bbox#91]
Hello and thanks for this nice lib!

I've found a path that gives me different results in the browser and with `svg-path-bbox`.

`svg-path-bbox` gives me a width of `302.40999999999997`

```js
import svgPathBbox from "svg-path-bbox"
const path =
  "M436.79 417.31C435.89 415.01 438.59 410.91 439.69 409.31C439.89 408.91 439.69 408.41 439.39 408.31C438.09 407.41 435.19 404.71 435.09 401.61C434.99 399.71 437.39 396.41 438.49 394.91C439.19 393.71 437.69 393.11 437.09 392.31C435.69 390.81 433.99 388.31 434.09 385.71C434.19 382.71 436.69 379.51 440.79 377.01C440.89 376.91 440.99 376.71 440.89 376.61C439.79 374.91 439.29 374.01 439.19 371.41C438.99 367.91 440.79 364.81 444.19 362.41C446.59 360.81 449.89 359.81 452.79 359.21C452.79 359.01 452.79 349.41 452.79 348.21V348.01V340.21C452.79 340.01 452.69 339.91 452.49 339.91L209.89 340.21C209.69 340.21 209.59 340.31 209.59 340.51L209.69 896.1H452.49C452.69 896.1 452.79 896 452.79 895.8V420.31C449.79 421.41 446.59 422.11 444.19 422.11C440.79 422.01 437.69 419.51 436.79 417.31Z "
const bbox = svgPathBbox(path)
const width = bbox[2] - bbox[0]

console.log(width) // 302.40999999999997
```

But for the same path, chrome gives me a width or `243.20001220703125`:

```js
document.body.innerHTML = `
  <svg width="800" height='800'><path fill='black' d="M436.79 417.31C435.89 415.01 438.59 410.91 439.69 409.31C439.89 408.91 439.69 408.41 439.39 408.31C438.09 407.41 435.19 404.71 435.09 401.61C434.99 399.71 437.39 396.41 438.49 394.91C439.19 393.71 437.69 393.11 437.09 392.31C435.69 390.81 433.99 388.31 434.09 385.71C434.19 382.71 436.69 379.51 440.79 377.01C440.89 376.91 440.99 376.71 440.89 376.61C439.79 374.91 439.29 374.01 439.19 371.41C438.99 367.91 440.79 364.81 444.19 362.41C446.59 360.81 449.89 359.81 452.79 359.21C452.79 359.01 452.79 349.41 452.79 348.21V348.01V340.21C452.79 340.01 452.69 339.91 452.49 339.91L209.89 340.21C209.69 340.21 209.59 340.31 209.59 340.51L209.69 896.1H452.49C452.69 896.1 452.79 896 452.79 895.8V420.31C449.79 421.41 446.59 422.11 444.19 422.11C440.79 422.01 437.69 419.51 436.79 417.31Z"/></svg>
`
const bbox = document.querySelector('path').getBBox()

console.log(bbox.width) // 243.20001220703125
```

**Comment by mondeja** [comment]
Thanks for the detailed report. It is fixed in [v1.2.1](https://github.com/mondeja/svg-path-bbox/releases/tag/v1.2.1).

**Comment by testerez** [comment]
Thanks for the quick fix @mondeja !!
